caasp-master.yaml

```diff
@@ -0,0 +1,116 @@
+heat_template_version: 2015-04-30
```

Comment:
should the version be moved or is this version the one that is needed: 
https://docs.openstack.org/heat/pike/template_guide/hot_spec.html#hot-spec-template-version

Reply:
good catch! that version string was for 'kilo', updated it to pike string '2017-09-01'